pyEDAA/ProjectModel/__init__.py

```diff
@@ -466,6 +473,30 @@ def SVVersion(self, value: SystemVerilogVersion) -> None:
 		self._svVersion = value
 
 
+@export
+class SystemRDLSourceFile(RDLSourceFile, HumanReadableContent):
+	"""A SystemRDL source file (of any language version)."""
+
+	_srdlVersion: SystemRDLVersion
+
+	def __init__(self, path: pathlib_Path, srdlVersion: SystemRDLVersion = None, project: 'Project' = None, design: 'Design' = None, fileSet: 'FileSet' = None):
+		super().__init__(path, project, design, fileSet)
+
+	@property
+	def SRDLVersion(self) -> SystemVerilogVersion:
+		"""Property setting or returning the SystemRDL version this SystemRDL source file is used in."""
+		if self._srdlVersion is not None:
+			return self._srdlVersion
+		elif self._fileSet is not None:
+			return self._fileSet.SRDLVersion
```

Comment:
This is not yet implemented on the fileset (and design) classes.

Reply:
I have now implemented it in the Design and FileSet classes. I have deleted the pySRDLModel Python module. And imported the SystemRDLVersion from the pySRDLModel Python package. This package doesn't exist yet and has to be created and released to PyPi before this pull request can be merged.

Reply:
A repository is prepared. See #47.